### ISSUE TITLE
Again update on json

### ISSUE THREAD

**Issue by sledgemhammer** [akamming/domoticz-coronadashboard#8]
Hi 

The plugin is broken, did not look in detail, but based on a quick review it seems like the format has changed once again

**Comment by akamming** [comment]
yeah just checked, indeed a changed json again. Currently don't have the time to work on this. Feel free to make a PR, i will merge...

**Comment by akamming** [comment]
Just checked the JSON.  there are quite a few changes, so i have to analyze and rewrite. Just decided that i will no longer maintain this plugin.  it is too much work to continuously compensate for the changing json. I will leayve the repository there, so if someone wants to take over, they can clone the repository

**Comment by sledgemhammer** [comment]
Many thanks for buiding the plugin and leaving it there.
I have no python skill’s : i tried to modify but only errors.
For now i disabled all but sewer, that seems still ok.

Totaly understandable to stop developping.
Again many thanks !
